.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 .DS_Store
+.nyc_output/
```

Comment:
The current invocation doesn't generate any files

Reply:
Oh, it might have been created by one of my failed attempts before I got it right.

Reply:
Nope, in creating a new PR I still see this directory being created, so I'll leave this.